### ISSUE TITLE
Blessed error with a hugo server command

### ISSUE THREAD

**Issue by Serabe** [ericwooley/thewooleyway#13]
When adding this command to a script:

```json
{
  "label": "hugo",
  "command": "make server",
  "screenConfig": {}
}
```

I get the following error:

```bash
$ node_modules/.bin/termax server
TypeError: Cannot read property 'slice' of null
    at /Users/serabe/programming/blog/node_modules/blessed/lib/program.js:2543:35
    at Array.forEach (native)
    at Program._attr (/Users/serabe/programming/blog/node_modules/blessed/lib/program.js:2542:11)
    at Box.Element._parseTags (/Users/serabe/programming/blog/node_modules/blessed/lib/widgets/element.js:498:26)
    at Box.Element.parseContent (/Users/serabe/programming/blog/node_modules/blessed/lib/widgets/element.js:393:22)
    at Box.Element.setContent (/Users/serabe/programming/blog/node_modules/blessed/lib/widgets/element.js:335:8)
    at Box.Element.insertLine (/Users/serabe/programming/blog/node_modules/blessed/lib/widgets/element.js:2383:8)
    at Box.Element.pushLine (/Users/serabe/programming/blog/node_modules/blessed/lib/widgets/element.js:2526:15)
    at PassThrough.<anonymous> (/Users/serabe/programming/blog/node_modules/@tww/terminus-maximus/lib/bufferDispay.js:98:16)
    at emitOne (events.js:115:13)
```

It is not solved if I set the command to the actual command being run (`.hugo/hugo server -t hugo-steam-theme -D -w --baseURL "//localhost:1313/`).

Let me know if I can help triaging this in any way.

Thank you for terminus maximus!

**Comment by ericwooley** [comment]
That is a strange error. I'll have to check it out. Unfortunately i'm on vacation this weekend (when I do most my FOSS work), but i'll check it out as soon as I get a chance.

**Comment by Serabe** [comment]
Thank you! Let me know if I can help you somehow.

**Comment by ericwooley** [comment]
Hey @Serabe, I'm looking into this, but I don't see any instructions on the hugo website for building with make. Is there an example anywhere I can look at for your current setup?

**Comment by Serabe** [comment]
[src-d/landing](https://github.com/src-d/landing) was the repo I was trying to add termmax to.

**Comment by Serabe** [comment]
Not the best repo, though. Let me try and create a smaller one.

**Comment by Serabe** [comment]
https://github.com/Serabe/terminus-maximus-bug

Reproduction: Hugo + Webpack. You can even take webpack out and still reproduces it.

**Comment by ericwooley** [comment]
Awesome, thanks! I should have enough time in the next day or two to dig in.

**Comment by ericwooley** [comment]
I'm not sure what it is about the line
`Watching for changes in /Users/ericwooley/projects/terminus-maximus-bug/{content,static}` that is breaking blessed. Opened an issue with blessed.

https://github.com/chjj/blessed/issues/308

**Comment by ericwooley** [comment]
Figured out that somehow the `{` and `}` were breaking it

**Comment by ericwooley** [comment]
fixed in 2.0.4

**Comment by Serabe** [comment]
Works. Thank you!
